src/components/FormTransaction/FormTransaction.jsx

```diff
@@ -44,52 +44,54 @@ function FormTransaction() {
           required
           style={{ color: "var(--accent-lightgrey)" }}
         />
-      </label>
 
-      <div className={css["transaction-select"]}>
-        <select
-          data-value={category}
-          required
-          value={category}
-          onChange={(e) => setCategory(e.target.value)}
-        >
-          <option value="">Product category</option>
-          <option value="Transport">Transport</option>
-          <option value="Products">Products</option>
-          <option value="Health">Health</option>
-          <option value="Alcohol">Alcohol</option>
-          <option value="Entertainment">Entertainment</option>
-          <option value="Housing">Housing</option>
-          <option value="Technique">Technique</option>
-          <option value="Communal, communication">
-            Communal, communication
-          </option>
-          <option value="Education">Education</option>
-          <option value="Other">Other</option>
-        </select>
-        <svg className={css["select-arrow"]} width="10" height="4">
-          <use href="/src/assets/icons.svg#icon-arrow"></use>
-        </svg>
-      </div>
-
-      <div>
-        <label className={css["calc-container"]}>
-          <input
-            className={css["calc-input"]}
+        <div className={css["transaction-select"]}>
+          <select
+            data-value={category}
             required
-            type="number"
-            value={amount}
-            placeholder="0,00"
-            onChange={(e) => setAmount(e.target.value)}
-          />
-          <svg className={css["calc-icon"]}>
-            <use href="/src/assets/icons.svg#icon-calculator"></use>
+            value={category}
+            onChange={(e) => setCategory(e.target.value)}
+          >
+            <option value="">Product category</option>
+            <option value="Transport">Transport</option>
+            <option value="Products">Products</option>
+            <option value="Health">Health</option>
+            <option value="Alcohol">Alcohol</option>
+            <option value="Entertainment">Entertainment</option>
+            <option value="Housing">Housing</option>
+            <option value="Technique">Technique</option>
+            <option value="Communal, communication">
+              Communal, communication
+            </option>
+            <option value="Education">Education</option>
+            <option value="Other">Other</option>
+          </select>
```

Comment:
wszystkie te oprions zrobił bym jako osobny plik który importujesz ale nie tutaj a do Hero expences i hero income jako zmienną tak jak to masz fajnie zrobione w przypadku tabeli gdzie w propsach podajesz co ma być wyświetlone
bo na chwilę obecną i expences i income mają taką samą rozwijaną listę. dla expences to jest ok ale income powinno mieć 2 opcje z tego co pamiętam, stały dochód i dodatkowe źródła czy jakoś tak

Reply:
dzięki, w ogóle nie zauważyłam , że są dwie różne rozwijalne listy :/